src/scripts/dungeons/Dungeon.ts

```diff
@@ -136,7 +136,7 @@ class Dungeon {
             if (typeof loot === 'string') {
                 return 1;
             } else {
-                return ((<DetailedLoot>loot).weight / (App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]() + 1) + 1) ?? 1;
+                return (10 ^ (<DetailedLoot>loot).weight / (App.game.statistics.dungeonsCleared[GameConstants.getDungeonIndex(DungeonRunner.dungeon.name)]() + 1) + 1) ?? 1;
```

Comment:
This might not do what you want it to do. (Hint: ^ is not exponentiation)

Reply:
True, I totally forgot about Math.pow!